compiler/src/lnntoamm/Types.ts

```diff
@@ -182,6 +238,12 @@ class Struct extends Type {
     super(name, ast);
     this.args = args;
     this.fields = fields;
+    this.order = {};
+    let sizeTracker = 0;
+    for (let fieldName in this.fields) {
+      this.order[fieldName] = sizeTracker;
+      sizeTracker += this.fields[fieldName].size();
```

Comment:
Wouldn't this *technically* blow up on us if we had a `void` field?

```ln
type Foo
  bar: void,
  baz: string,
}
```

would probably do a *lot* of weird crap if you tried to access either field.

Reply:
I think it'll be UB to access `bar` but not `baz` - I'm not worried about managing this specific case yet since some things related to `void` might have to change once conditionals are implemented